report/how_to.md

```diff
@@ -0,0 +1,70 @@
+# User Guide
+
+## How to use this book
+
+### Validation
+
+TODO
+
+### Verification
+
+Each verification case was created using either the [`MES`](./verification/methods.md/#method-of-exact-solutions-mes) or [`MMS`](./verification/methods.md/#method-of-manufactured-solutions-mms) and are tagged accordingly.
+
+TODO
+
+### Navigating tags
+
+Each case is tagged and tag's cases can be navigated through the [🏷 Tags](./_tags/tagsindex.md) index.
+
+* [1D](./_tags/1d.md) : One-dimensional simulations.
+* [2D](./_tags/2d.md) : Two-dimensional simulations.
+* [`MES`](./_tags/mes.md) : Cases using the [`MES`](./verification/methods.md/#method-of-exact-solutions-mes)
+* [`MMS`](./_tags/mms.md) : Cases using the [`MMS`](./verification/methods.md/#method-of-manufactured-solutions-mms).
+* ...
+
+### Executing and editing code
+
+Press the Live Code button on the toolbar to edit and run the code. \
+![alt text](./how_to_media/live_code_button.png)
```

Comment:
I wonder if we can find this logo online instead of storing it locally.

Reply:
think not, they're css generated from [this site](https://fontawesome.com/)

Reply:
Is this overkill?

Reply:
not at all, thanks for the find